gammapy/modeling/models/tests/test_temporal.py

```diff
@@ -107,6 +111,24 @@ def test_time_sampling():
     assert_allclose(sampler_uniform.value, [1261.65802564, 6026.9299098], rtol=1e-5)
 
 
+def test_ConstantTemporalModel():
+    norm = 10.0
+    temporal_model = ConstantTemporalModel(norm)
+
+    t_ref = "2010-01-01T00:00:00"
+    t_min = "2010-01-01T00:00:00"
+    t_max = "2010-01-01T08:00:00"
+
+    sampler = temporal_model.sample_time(
+        n_events=2, t_min=t_min, t_max=t_max, random_state=0, t_delta="10 min"
+    )
+
+    sampler = u.Quantity((sampler - Time(t_ref)).sec, "s")
+
+    assert len(sampler) == 2
+    assert_allclose(sampler.value, [15805.82891311, 20597.45375153], rtol=1e-5)
+
```

Comment:
Please add a short test for `.evaluate_norm_at_time()`

Reply:
Ok.